### PR TITLE
[wallet/desktop] feat: split button component unit test

### PR DESCRIPTION
**Pull request by AnthonyLaw**
## What was the issue?
- missing unit test in the `splitButton` component.

## What's the fix?
- Added unit test  `splitButton` component.

### CODE REVIEW

**Comment by AnthonyLaw** [comment]
I will remove `SplitButton` because have not been used anywhere

**Comment by yilmazbahadir** [comment]
I think it might be a useful component, I'd keep it for now.

**Comment by AnthonyLaw** [comment]
> I think it might be a useful component, I'd keep it for now.

alright, let me restore it 👍🏼

File: __tests__/components/SplitButton.spec.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            },
+        ]);
+
+        await vm.$nextTick();
+
+        // Act + Assert:
+        const dropdownComponent = wrapper.findAll('dropdown-item-stub');
+        const dropdownComponentIcon = wrapper.findAll('icon-stub');
+
+        expect(dropdownComponent.length).toBe(2);
```
Comment (Jaguar0625) [inline __tests__/components/SplitButton.spec.ts:70]: you should also check dropdownComponentIcon length is 2

**Review by Jaguar0625 — APPROVED** [review]

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**